### PR TITLE
add call fields for Twilio integration

### PR DESCRIPTION
**Pull request by zalary**
it has been a while since I did this last, please forgive and let me know if I'm missing something obvious.

### CODE REVIEW

File: fields/call.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ "id": "call.forwarded_to_phone",  "name": "Call Forwarded to Phone",   "type": "phone",  "description": "Additional information from LowerMyBills regarding the lead" }
+{ "id": "call.forwarded_from",      "name": "Call Forwarded from Phone", "type": "phone",  "description": "This parameter is set only when a forwarded call is received, but its value depends on the caller's carrier including information when forwarding. Not all carriers support passing this information."}
+{ "id": "call.called_phone",        "name": "Called Phone",              "type": "phone",  "description": "The number called.  The tracking number for inbound calls. " }
+{ "id": "call.status",              "name": "Call Status",               "type": "string", "description": "A descriptive status for the call. Values such as queued ,ringing, in-progress, completed, busy, failed, no-answer, or canceled." }
+{ "id": "call.direction",           "name": "Call Direction",            "type": "string", "description": "Direction of the call - inbound or outbound." }
+{ "id": "call.duration",            "name": "Call Duration",             "type": "number", "description": "Duration in seconds of the completed call." }
```
Comment (alexkwolfe) [inline fields/call.json]: I recommend `examples` for all the non-phone types.

File: fields/call.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{ "id": "call.forwarded_to_phone",  "name": "Call Forwarded to Phone",   "type": "phone",  "description": "The phone number the Twilio number is set to forward to.  This information is provided via the querystring in the Twilio callbackurl" }
+{ "id": "call.forwarded_from",      "name": "Call Forwarded from Phone", "type": "phone",  "description": "This parameter is set only when a forwarded call is received, but its value depends on the caller's carrier including information when forwarding. Not all carriers support passing this information."}
+{ "id": "call.called_phone",        "name": "Called Phone",              "type": "phone",  "description": "The number called.  The tracking number for inbound calls. " }
+{ "id": "call.status",              "name": "Call Status",               "type": "string", "description": "A descriptive status for the call. Values such as queued ,ringing, in-progress, completed, busy, failed, no-answer, or canceled.", "examples": ["queued", "ringing", "in-progress", "completed", "busy", "failed", "no-answer", "canceled"] }
```
Comment (alexkwolfe) [inline fields/call.json]: You've got a out-of-place space character in this one: `queued ,ringing`.  Also, I think it's safe to leave the examples out of the description. The examples might not show in the UI, but that's something that needs to be fixed there. Other field examples have been removed from `description` in favor of including them just in the `examples`.

**Comment by alexkwolfe** [comment]
No further comments. LGTM. :rocket:

File: fields/call.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{ "id": "call.forwarded_to_phone",  "name": "Call Forwarded to Phone",   "type": "phone",  "description": "The phone number the Twilio number is set to forward to.  This information is provided via the querystring in the Twilio callbackurl" }
+{ "id": "call.forwarded_from",      "name": "Call Forwarded from Phone", "type": "phone",  "description": "This parameter is set only when a forwarded call is received, but its value depends on the caller's carrier including information when forwarding. Not all carriers support passing this information."}
```
Comment (alexkwolfe) [inline fields/call.json]: Damnit. I do have one more. I wonder if this should be `call.forwarded_from_phone` for consistency with the `call.forwarded_to_phone`?

Comment (zalary) [inline fields/call.json]: or should we change forwarded_to_phone to just forwarded_to   it's already phone type?

**Pull request merged**